src/service/video-manager-impl.js

```diff
@@ -894,13 +945,235 @@ class VideoEntry {
     // Update docking state
     if (this.scrollMap_(DOCK_SCALE, 1) == DOCK_SCALE) {
       this.dockState_ = DockStates.DOCKED;
+      this.initializeDragging_();
+      this.drag_();
```

Comment:
It needs to run every RAF for the momentum drag to kick in (so that it moves the element even after the user is no longer dragging until the velocity is null).